src/network/esp/include/DefaultNetworkConfig.in.h

```diff
@@ -0,0 +1,16 @@
+#ifndef __DEFAULTNETWORKCONFIG_H__
+#define __DEFAULTNETWORKCONFIG_H__
+// clang-format off
+
+#include <logger/ILogger.h>
+
+/**
+ * @brief Template file used by CMake to provide default or user-provided values for the network configuration
+ */
+
+#define DEFAULT_SSID "@SSID@"
+#define DEFAULT_PASSWORD "@PASSWORD@"
+#define DEFAULT_AUTH_MODE @AUTH_MODE@
```

Comment:
Document those in the readme, in particular the different values of auth mode or put a link to the ESP doc